crud/registry.go

```diff
@@ -23,11 +23,11 @@ func (r *Registry) typesMap() map[Kind]Actions {
 // An error will be returned if kind was previously registered.
 func (r *Registry) Register(kind Kind, a Actions) error {
 	if kind == "" {
-		return errors.New("kind cannot be empty")
+		return fmt.Errorf("kind cannot be empty")
 	}
 	m := r.typesMap()
 	if _, ok := m[kind]; ok {
-		return errors.New("kind '" + string(kind) + "' already registered")
+		return fmt.Errorf("kind '" + string(kind) + "' already registered")
 	}
```

Comment:
Would it be an opportunity to use verbs here and in the rest of this file ?

Reply:
Can we do this as a follow up PR rather than this file?
I really wanted to simplify only one aspect of code with this PR and I'd like to avoid scope creep here.

Reply:
Ok!